pkg/state/state_macvlan_network_test.go

```diff
@@ -0,0 +1,144 @@
+/*
+Copyright 2024 NVIDIA
```

Comment:
Please change to:
2024 NVIDIA CORPORATION & AFFILIATES

Reply:
Done